Create jessales

jessales opened the pull request:
# Application to the Class of 2022🎓

This pull request template helps you complete an application to the **Class of 2022**. Use the checklist below to verify you have followed the instructions correctly. 

## Checklist ✅

- [x] I have read the instructions on the README file before submitting my application. 
- [x] I made my submission by creating a folder on the `_data` folder and followed the naming convention mentioned in the instructions (`<username>`) and markdown file.
- [x] I have submitted a [swag shipping form](https://airtable.com/shrVMo8ItH4wjsO9f).
- [x] I have used the Markdown file template to add my information to the Year Book.
- [x] I understand that a reviewer will merge my pull request after examining it or ask for changes in case needed.
- [x] I understand I should not tag or add a reviewer to this Pull Request.
- [x] I have [added the event](https://calendar.google.com/calendar/u/0/r/eventedit?dates=20220611T200000Z/20220611T220000Z&details&location=https://www.twitch.tv/githubeducation&text=%F0%9F%8E%89%F0%9F%8E%8A+GitHub+Graduation+2022+%F0%9F%8E%89%F0%9F%8E%8A) to my Calendar.

Commit:
jessica-sales.md

Commit:
Created jessica-sales

github-education reviewed (changes requested):
**Hi jessica-sales,**
**Welcome to graduation!**


### I have a few items I need you to take care of before I can merge this PR:

* *Uh Oh! I've found some issues with where you have created your files!* 
	*Please ensure all changes are contained within the `_data/jessica-sales/` directory. Invalid file paths:* 

	* `_dados/jessica-sales.md`


Feel free to re-request a review from me and I'll come back and take a look!

Commit:
Adicionar meu perfil ao repositório

Assinado por:  Jessica Sales <jessica.sales21@outlook.com>

Commit:
Adicionando meu nome no repositório 

Adicionando meus dados no repositório do Github 

e-mail <jessica.sales21@outlook.com>

Commit:
Delete jessica-sales.md

Commit:
Delete jessica-sales.md

github-education reviewed (changes requested):
**Hi jessales,**
**Welcome to graduation!**


### I have a few items I need you to take care of before I can merge this PR:

* *I'm not seeing a valid student developer pack approval, please submit an [application](https://education.github.com/discount_requests/student_application?utm_source=2021-06-05-GitHubGraduation) to get started*
* *It looks like you still need to fill out the [shipping form](https://airtable.com/shrVMo8ItH4wjsO9f) to continue*
* *Uh Oh! I've found some issues with where you have created your files!* 
	*Please ensure all changes are contained within the `_data/jessales/` directory. Invalid file paths:* 

	* `_dados/jessales.md`

* *You need to agree to our COC on the shipping form pretty please!*

Feel free to re-request a review from me and I'll come back and take a look!

Commit:
Created jessales.md

Adicionar meus dados

Commit:
Participação da Graduação GitHub
